src/main/webapp/vendor/startbootstrap/saved-planner-content.jsp

```diff
@@ -0,0 +1,15 @@
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix ="c" %> 
+<%@ taglib uri="http://java.sun.com/jsp/jstl/fmt" prefix ="fmt" %>
+<!-- Begin Page Content -->
+<div class="container-fluid">
+  <!-- Page Heading -->
+  <div class="d-sm-flex align-items-center justify-content-between mb-4" id="alert-container">
+    <h1 class="h3 mb-0 text-gray-800">Planner: My Saved Plans</h1>
+  </div>
+  <div class="row">
+    <!-- Course Selection Area -->
+    <div class="col-xl-12 col-lg-7" id= "plan-column">
```

Comment:
are the other 5 width units taken up by the sidebar in `lg-7`? What's the rationale here?

Reply:
I'm actually not sure, it seems to look the same when i remove it from the class -- I was following the pattern set in the UI templates, but I think i'll just remove it.

Reply:
Ah ok. Normally this changes based on your screen width (if you resize your window, the elements' widths and positions may shift around). Probably a bit beyond the scope of the work here, but it's good to learn someday.